content/java/get-started.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: "Get Started"
+title: "WorldWind Java Get Started"
 date: 2017-07-27T09:50:17-05:00
 draft: false
 ---
 
```

Comment:
The numbered lists don't render in the correct order, and their formatting isn't as good a fit for the new Hugo site. See https://goworldwind.org/getting-started/ for the original intended formatting.